resolwe/flow/managers/dispatcher.py

```diff
@@ -193,9 +193,9 @@ def __init__(self):
     class _SettingsManager(object):
         """Context manager for settings overrides.
 
-        Because Django's :meth:`~django.test.override_settings` is a
-        context manager, it would make the code awkward if the manager's
-        support for this wasn't also in the form of a context manager.
+        Because Django's ``django.test.override_settings`` is a context
```

Comment:
I'm curious why did you remove the `:meth:` part?

I noticed that this is not included in the docs since `_SettingsManager` is a private class, hence is not really important, but still...

Reply:
Sphinx was complaining about missing reference targets so I took the easy way out because Django objects are external to Resolwe; Sphinx was probably expecting to be able to resolve it with objects within the local codebase. I can look into this a bit more.

Reply:
I figured out what was wrong, we tried to reference a method, but apparently, Sphinx considers `override_settings` to be function.

Thus, changing ``:meth:`~django.test.override_settings` `` to `` func:`django.test.override_settings` `` works like a charm.